stable/awsebscsiprovisioner/templates/storageclass.yaml

```diff
@@ -21,3 +21,7 @@ parameters:
   {{- if .Values.storageclass.encrypted }}
   encrypted: {{ .Values.storageclass.encrypted | quote }}
   {{- end }}
+{{- with .Values.storageclass.allowedTopologies }}
+allowedTopologies:
+{{ toYaml . }}
```

Comment:
Just checking, should this line be indented?

Reply:
The content will be an array of `topologySelectorTerms` which should not be indented.
``` 
allowedTopologies:
- matchLabelExpressions:
  - key: topology.ebs.csi.aws.com/zone
     values:
     - us-west-2a
     - us-west-2b
     - us-west-2c
- nodeSelector:
  ...
```